setup/setup_snyk.sh

```diff
@@ -19,31 +19,24 @@ die () {
     exit 1
 }
 
-[ "$#" -eq 2 ] || die "Setup Snyk requires two argument, $# provided"
+# Check if correct number of arguments is provided
+[ "$#" -eq 2 ] || die "Setup Snyk requires two arguments, $# provided"
 
 cd "$(mktemp -d)"
-
 echo "Installing the $1 version of Snyk on $2"
 
 VERSION=$1
-BASE_URL="https://static.snyk.io/cli"
+MAIN_URL="https://downloads.snyk.io/cli"
+BACKUP_URL="https://static.snyk.io/cli"
```

Comment:
issue: we'll need to add the utm parameter to each request, adding it here might be the best place.